packages/gatsby-source-wordpress/README.md

```diff
@@ -83,6 +83,11 @@ plugins: [
       // Set verboseOutput to true to display a verbose output on `npm run develop` or `npm run build`
       // It can help you debug specific API Endpoints problems
       verboseOutput: false,
+      // Search and Replace Urls across WordPress content
+      searchReplace: {
```

Comment:
Let's make this option `searchAndReplaceURLs` — no reason not to be explicit :-)